plugins/thread.py

```diff
@@ -54,7 +55,10 @@ def run(self):
 
             # generate production archive
             self.progress(75)
-            temp_file = self.process_manager.generate_archive(temp_dir, temp_file)
+            temp_file = self.process_manager.generate_archive(temp_dir + "_g", temp_file)
+            shutil.move(temp_file, temp_dir)
+            shutil.rmtree(temp_dir + "_g")
```

Comment:
Can you move this line after line 112 in finally block?
I think it best that the cleanup will be together at the end of the process.

Reply:
trying to move the archive file out of the _g folder, then delete it so it doesnt get copied to output_path

Reply:
I understand. Moving ` shutil.rmtree(temp_dir + "_g")` to the end will result in the same manner. Never mind, it is my just code styling.